Create komplete-complete.txt

ScruffyInLondon opened the pull request:
After opening up Native Access for the first time, and logging in, I checked to see what was in ```/Libarary/Application Support/Native Access/Service Center``` and found a single file named ```NativeAccess.xml```.

I've parsed this with ```xmlstarlet``` and managed to pull out this list of products from it, which appear to match up with your known ProductName/UPID lists and the ones I have.

I believe this could be their entire product inventory as of today.

Commit:
Create komplete-complete.txt

Complete list of NI products.

geoffklee commented:
This is great! Could you add a few lines of comments to the start of the file indicating how you obtained it, and when, so it makes sense to future readers? Thanks very much.

Commit:
Update komplete-complete.txt

Date obtained, Native Access version, and details on how to find this information.

ScruffyInLondon commented:
Not a problem, I've updated it with the requested information.

geoffklee reviewed (approved):
Thanks!
